exercises/supervised-classification/ex_rnw/ex_research-group-logreg.Rnw

```diff
@@ -0,0 +1,48 @@
+Chapter 3.4 of the lecture introduces \textbf{logistic regression}. The medical research group from last week is eager to find out if it can be used to predict whether a patient admitted to the hospital will require intensive care. This is a binary classification task with target space $\Yspace = \setzo$, with $y=1$ if the patient requires intensive care and $y=0$ if not. The feature space is the same as before:
```

Comment:
"The lecture.." Nummerierung kann sich mal ändern und dann stimmt der verweis nicht mehr.